src/Lykke.Service.OperationsCache/Settings/ServiceSettings/OperationsCacheSettings.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Lykke.Service.OperationsCache.Settings.ServiceSettings
+{
+    public class OperationsCacheSettings
+    {
+        public string CacheInstanceName { get; set; }
```

Comment:
Usually, in this section the Radis section is registered and there is a link to the access point and the name of the instance

Not understand for what we add separte section RedisSettings